config/kernel-userns-capabilities.m4

```diff
@@ -56,6 +56,7 @@ AC_DEFUN([ZFS_AC_KERNEL_SRC_CRED_USER_NS], [
 	],[
 		struct cred cr;
```

Comment:
```suggestion
		struct cred cr __attribute__ ((unused));
```

This works, but we've tried to use the `__attribute__ ((unused))` in the m4 config files instead of `(void)cr`.

Same change in `ZFS_AC_KERNEL_SRC_GROUP_INFO_NBLOCKS`